tests/system/test_root_metronome.py

```diff
@@ -327,6 +327,7 @@ def test_metronome_shutdown_with_no_extra_tasks():
 def setup_module(module):
     common.wait_for_metronome()
     common.wait_for_cosmos()
+    common.cluster_info()
```

Comment:
is this just to block until ready or something? It seems like we're calling cluster_info and discarding the value, and that's not clear why we'd do that.